Documentation/security/threat-model.rst

```diff
@@ -0,0 +1,751 @@
+|                 | remains unaffected.   |                            |
++-----------------+-----------------------+----------------------------+
+| Observability   | None                  | Cilium’s Hubble flow-event |
+| data            |                       | observability can be used  |
+|                 |                       | to provide reliable audit  |
+|                 |                       | of the attacker’s L3/L4    |
+|                 |                       | and L7 network             |
+|                 |                       | connectivity. Traffic sent |
+|                 |                       | by the attacker will be    |
+|                 |                       | attributed to the worker   |
+|                 |                       | node, and not to a         |
+|                 |                       | specific Kubernetes        |
+|                 |                       | workload.                  |
++-----------------+-----------------------+----------------------------+
+
+Recommended Controls
+^^^^^^^^^^^^^^^^^^^^
+
+In addition to the recommended controls against the :ref:`kubernetes-workload-attacker`:
+
+-  Container images should be regularly patched to reduce the chance of
+   compromise
+-  Minimal container images should be used where possible
+-  Host-level privileges should be avoided where possible
+-  Ensure that the container users do not have access to the underlying
+   container runtime
+
+.. _root-equivalent-host-attacker:
+
+Root-equivalent Host Attacker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A “root” privilege host attacker has full privileges to do everything on
+the local host. This access could exist for several reasons, including:
+
+-  Root SSH or other console access to the Kubernetes worker node.
+-  A containerized workload that has escaped the container namespace as
+   a privileged user.
+-  Pods running with ``privilege: true`` or other significant
+   capabilities like ``CAP_SYS_ADMIN`` or ``CAP_BPF``.
+
+.. image:: images/cilium_threat_model_root.png
+
+.. rst-class:: wrapped-table
+
++-------------------+--------------------------------------------------+
+| **Threat          | **Identified STRIDE threats**                    |
+| surface**         |                                                  |
++===================+==================================================+
+| Cilium agent      | In this situation, all potential attacks covered |
+|                   | by STRIDE are possible. Of note:                 |
+|                   |                                                  |
+|                   | -  The attacker would be able to disable eBPF on |
+|                   |    the node, disabling Cilium’s network and      |
+|                   |    runtime visibility and enforcement. All       |
+|                   |    further operations by the attacker will be    |
+|                   |    unlimited and unaudited.                      |
```

Comment:
I believe that this is accurate for the "root-equivalent Host attacker", though @jrfastab may have some ideas about ways to lock down & audit such cases as well.

(no change necessary at this time, just dropping an FYI)